config/services.yaml

```
@@ -6,6 +6,18 @@
 parameters:
 
 services:
+  app.customer_type.value_search_filter:
+    parent: 'api_platform.doctrine_mongodb.odm.search_filter'
+    arguments:
+      - 'value': 'exact'
+    tags:
+      - { name: 'api_platform.filter', id: 'customer_value.mongodb.value_filter' }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)